mlserver/settings.py

```diff
@@ -241,6 +241,12 @@ class Config:
     _custom_metrics_server_settings: Optional[dict] = None
     _custom_grpc_server_settings: Optional[dict] = None
 
+    cache_enabled: Optional[bool] = False
+    """Enable Caching for the model predictions."""
+
+    cache_size: Optional[int] = 100
+    """Cache size to be used if caching is enabled."""
```

Comment:
Given these already get a default non-`None` value, should we remove the `Optional` annotation?

Reply:
Updated in [43dfc1c](https://github.com/SeldonIO/MLServer/pull/1440/commits/43dfc1c7587e738d377d437761fcb2bcade94854)